pykit/types.py

```diff
@@ -0,0 +1,157 @@
+class Parameter:
+    def __init__(self, parameterType, direction='inout', byValue=False, isReturn=False):
+        """
+        :param name:
+        :param value:
+        """
+        self.parameterType = parameterType
+        self.direction = direction
+        self.isReturn = isReturn
+        self.byValue = False
+        self.payload = {}
+    
+    def get_payload(self):
+        self.payload = self.parameterType.get_payload()
+        if self.isReturn:
+            self.payload['by'] = 'return'    
+        if self.byValue:
+            self.payload['by'] = 'val'    
+        
+        return self.payload
+    
+
+class ParameterType:
+    """
+    Object for IBM i ParameterTypes.
+    """
+    def __init__(self, name, value):
+        """
+        :param name:
+        :param value:
+        """
+        self.value = value
+        self.name = str(name)
+        self.payload = {"name":self.name}
+            
+            
+    def get_payload(self):
+        self.payload['value'] = self.value        
+        return self.payload
+    
+class Float(ParameterType):
+    """
+    Object for IBM i Float.
+    """
+    def __init__(self, name, length, precision, value):
```

Comment:
Can a float have a precision? I think it has to be specified as 0. If so, just like the `Integer` class, leave it off the parameter list.